web/annotation_layer_builder.css

```
@@ -237,6 +249,18 @@
   width: 100%;
 }
 
+.annotationLayer .fileAttachmentAnnotation .popupTriggerArea {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.annotationLayer .fileAttachmentAnnotation .popupWrapper {
```

This looks like unnecessary duplication, can't we just do the following instead?
```suggestion
.annotationLayer .fileAttachmentAnnotation .popupTriggerArea,
.annotationLayer .fileAttachmentAnnotation .popupWrapper {
```

Also, what's "special" about the FileAttachmentAnnotation that required these changes?